stubs/stripe/stripe/api_resources/test_helpers/test_clock.pyi

```diff
@@ -0,0 +1,11 @@
+from _typeshed import Self
+from typing import Any, Literal
+
+from stripe.api_resources.abstract import CreateableAPIResource, DeletableAPIResource, ListableAPIResource
+
+class TestClock(CreateableAPIResource, DeletableAPIResource, ListableAPIResource):
+    OBJECT_NAME: Literal["test_helpers.test_clock"]
+
+    @classmethod
+    def advance(cls: type[Self], idempotency_key: str | None = ..., **params: Any) -> Self: ...
```

Comment:
This weird thing Stripe does where the method is both an instance method and a class method is confusing, but what you currently have is invalid.

I think putting only a classmethod would produce correct types, because classmethods can also be called on instances. So I suggest removing the last line.

Reply:
ah thank you! removing the instance method worked like a charm!